modules/apps/frontend-js/frontend-js-react-web/npmscripts.config.js

```diff
@@ -20,5 +20,14 @@ module.exports = {
 			},
 			ignore: ['**/legacy/config.js'],
 		},
+		exports: {
+			'classnames': 'classnames/index',
+			'formik': 'formik/dist/index',
+			'prop-types': 'prop-types/index',
+			'react': 'react/index',
+			'react-dnd': 'react-dnd/dist/cjs/index',
+			'react-dnd-html5-backend': 'react-dnd-html5-backend/dist/cjs/index',
+			'react-dom': 'react-dom/index',
```

Comment:
Is it possible to infer the entry point via the modules package.json's `main`? Not a deal breaker, but just curious

Reply:
Yeah, maybe we can do that, but sometimes we may want to export other inner modules too (though I admit that practice should disappear with the time...). If we want to do that, we need to implement it in `npm-scripts`...